### PR TITLE
Pull request template

### PR DESCRIPTION
**Pull request by ewandennis**
# THIS PR WILL BE MERGED ON 2019-02-21 12.00 EST

# What Changed
Added a PR template to give people a minimum of info when picking up a review, and lighten up the task of opening a PR.

Once merged, the contents of this template will appear pre-populated in the "Open a pull request" form in GitHub.

# How To Test
Preview the template in GitHub or a Markdown viewer.
Critique the sections, content and most importantly guidance.

# Todo
- [x] Address review feedback

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# What Changed
+ - What did you do and why? (Alignment between PR commits and this section helps your reviewers)
```
Comment (rajumsys) [inline .github/pull_request_template.md]: What we are going to cover in "why" section? is it something specific to implementation or general (like why this PR/feature/fix exist?)

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# What Changed
```
Comment (jonambas) [inline .github/pull_request_template.md]: h1s are a little big 🙂

File: .github/pull_request_template.md (hunk 1 above)
Comment (bkemper) [inline .github/pull_request_template.md]: How about a description like...

> What did you do to satisfy the AC?  Screenshots are welcome.

**Comment by kchu1337** [comment]
Maybe add a section for the Acceptance Criteria of the ticket? Could also be included in the `What Changed` section.

**Pull request merged**